checkers/testdata/dupOption/positive_tests.go

```diff
@@ -0,0 +1,62 @@
+package checker_test
+
+import "context"
+
+type panel struct {
+	name   string
+	width  int
+	height int
+}
+
+func newPanel(name string, opts ...func(c *panel)) *panel {
+	cfg := &panel{
+		name:   name,
+		width:  0,
+		height: 0,
+	}
+	for _, opt := range opts {
+		opt(cfg)
+	}
+
+	return cfg
+}
+
+func withWidth(w int) func(c *panel) {
+	return func(c *panel) {
+		c.width = w
+	}
+}
+
+func withHeight(h int) func(c *panel) {
+	return func(c *panel) {
+		c.height = h
+	}
+}
+
+type exportOptions struct{}
+
+type exportOpt func(context.Context, *exportOptions) error
```

Comment:
Can you add a similar case but where `exportOpt` is an alias, literally:

```
type exportOpt = func(context.Context, *exportOptions) error
```

Supporting aliases is often buggy in linters :(